jest.setup.ts

```diff
@@ -0,0 +1,6 @@
+import { waffleJest } from "@ethereum-waffle/jest";
```

Comment:
This file is not needed here

Reply:
We use it line 4 with 'expect.extend(waffleJest);', should I delete this line too ?